src/storage/disk_manager.c

```diff
@@ -221,6 +221,18 @@ static DKNSECTS disk_Temp_max_sects = -2;
 typedef UINT64 DISK_STAB_UNIT;
 #define DISK_STAB_UNIT_SIZE_OF sizeof (DISK_STAB_UNIT)
 
+/*
+ * Previously, when a user entered excessively long remarks, the remarks would be trimmed to avoid exceeding the page size. 
+ * This trimming could occur at unwanted positions, potentially removing important content. 
+ * To address this, we have removed the trimming functionality and defined a maximum length for remarks. 
+ * Now, remarks longer than this defined maximum length will be restricted from being entered. 
+ * 
+ * DISK_VOLUME_HEADER = fixed fields + variable fields
+ * variable fields = volume fullname + next volume fullname + remarks
+ * volume fullname & next volume fullname max length = DB_MAX_PATH_LENGTH (include null charactor)
```

Comment:
TODO 도 있으면 좋을 것 같습니다. 아래는 예시입니다. 희수님의 배경지식을 토대로 TODO를 작성해주시면 좋을 것 같네요. 문제 상황, 개선 사항, 언제 개선할지 등을 적어주시면 좋겠습니다. 물론 한글로 적으면 안됩니다. 
```suggestion
 * volume fullname & next volume fullname max length = DB_MAX_PATH_LENGTH (include null charactor)
 * TODO:
 *  DB_PAGESIZE가 16K 미만일 때는 고려되어있지 않다. 이 부분을 포함한 모든 구현부에서 고려하고 있지 않기 때문에, 추후에 DB_PAGESIZE에 따라 각 멤버의 길이를 조정하는 이슈를 진행할 때 이 계산식도 수정이 필요하다. 
```

Reply:
추가하였습니다.